src/github.js

```diff
@@ -133,23 +136,42 @@ export default class GitHub {
 			issue_number: prNumber,
 		};
 
-		const commentMessage =
-		"Here is a list of everyone that appears to have contributed to this PR and any linked issues:\n\n" +
+		let commentMessage = "The following accounts have interacted with this PR and/or linked issues. I will continue to update these lists as activity occurs. You can also manually ask me to refresh this list by adding the `props-bot` label.\n\n";
+
+		if ( contributorsList['unlinked'].length > 0 ) {
+			commentMessage += "## Unlinked Accounts\n\n" +
+				"The following contributors have not linked their GitHub and WordPress.org accounts: @" + contributorsList['unlinked'].join(', @') + ".\n\n" +
+				"Contributors, please [read how to link your accounts](https://make.wordpress.org/core/2020/03/19/associating-github-accounts-with-wordpress-org-profiles/) to ensure your work is properly credited in WordPress releases.\n\n";
+		}
+
+		commentMessage += "## Core SVN\n\n" +
```

Comment:
@desrosj, would it be possible to make this configurable? While `Core SVN` makes perfect sense in WordPress develop, the `GitHub Merge` isn't that useful. Then, when contributing to Gutenberg, it exactly the other way around. So while the default could be as is, what if you could pass the setting when defining the GitHub workflow?

```
svnProps: true
gitCoAuthored: true
```

Reply:
Could you make an issue for this on the @WordPress/props-bot-core repo?

Reply:
Pleasure: https://github.com/WordPress/props-bot-action/issues/38 :)